src/app/cli/src/init/client.ml

```diff
@@ -873,14 +873,14 @@ let dump_ledger =
         ~doc:"STATE-HASH State hash (default: best state hash)"
         (optional string))
   in
-  let json_flag = Cli_lib.Flag.json in
-  let flags = Args.zip2 sl_hash_flag json_flag in
+  let plaintext_flag = Cli_lib.Flag.plaintext in
+  let flags = Args.zip2 sl_hash_flag plaintext_flag in
   Command.async ~summary:"Print the ledger with given Merkle root"
```

Comment:
Could you change the description to `Print the staged-ledger corresponding to the given state-hash ( best-tip by default)` or something like that? The command doesn't accept merkle root

Reply:
Updated docstring, see if it's OK.